src/components/loader/loader.spec.tsx

```diff
@@ -1,36 +1,57 @@
 import React from "react";
-import TestRenderer from "react-test-renderer";
-import { shallow } from "enzyme";
+import { mount } from "enzyme";
 import {
   assertStyleMatch,
   testStyledSystemMargin,
 } from "../../__spec_helper__/test-utils";
-import Loader, { LoaderProps } from ".";
+import Loader from ".";
 import StyledLoader from "./loader.style";
 import StyledLoaderSquare from "./loader-square.style";
+import useMediaQuery from "../../hooks/useMediaQuery";
 
-function renderLoader(props: LoaderProps = {}) {
-  return shallow(<Loader {...props} />);
-}
-
-function renderStyles(props = {}) {
-  return TestRenderer.create(<StyledLoader {...props} />);
-}
+jest.mock("../../hooks/useMediaQuery", () => {
+  return {
+    __esModule: true,
+    default: jest.fn().mockReturnValue(false),
+  };
+});
 
 describe("Loader", () => {
-  testStyledSystemMargin((props) => <Loader {...props} />);
-
-  it("renders as expected", () => {
-    assertStyleMatch(
-      {
-        textAlign: "center",
-      },
-      renderStyles().toJSON()
-    );
+  describe("margins", () =>
```

Comment:
Not something to change but we should probably review the text for the styled system utils so we don't need to add the `describe("margins")...` as the output still reads a little funny, but it's better than before etc so can be left as you have it for now